pkg/engine/internal/imageverifier.go

```diff
@@ -443,6 +444,12 @@ func (iv *ImageVerifier) verifyAttestations(
 					image = imageInfo.String()
 				}
 
+				iv.logger.V(2).Info("verifying image signatures", "image", image, "attestors", len(imageVerify.Attestors), "attestations", len(imageVerify.Attestations))
+				if err := iv.policyContext.JSONContext().AddImageInfo(imageInfo, cfg); err != nil {
+					iv.logger.Error(err, "failed to add image to context")
+					return engineapi.RuleError(iv.rule.Name, engineapi.ImageVerify, fmt.Sprintf("failed to add image to context %s", image), err), ""
+				}
+
```

Comment:
what is this for?

Reply:
as we need to add the payload for the FetchAttesstation in policy context, I have added this. Is this right?